src/tools/rbd_nbd/rbd-nbd.cc

```diff
@@ -752,41 +812,31 @@ class NBDWatchCtx : public librbd::UpdateWatchCtx
     , io_ctx(_io_ctx)
     , image(_image)
     , size(_size)
-  { }
+  {
+    handle_notify_thread = make_named_thread("rbd_notify",
```

Comment:
Should the thread be renamed to "rbd_handle_notify" too then?

Reply:
sure